docs/README.md

```diff
@@ -37,44 +39,44 @@ Or to a github branch (e.g. `reference_proto`) of any public github repository (
 https://www.cyberbotics.com/doc/guide/index?version=remidhum:reference_proto
 ```
 
-## Run the doc locally
+## To Run the doc locally,follow these steps:
 
-Set the terminal to the 'docs' directory.
+1. Set the terminal to the 'docs' directory.
 
 ```sh
 cd $WEBOTS_HOME/docs
 ```
 
-Create or update the `index.html` page:
+2. Create or update the `index.html` page:
 
 ```sh
 python local_exporter.py
 ```
 
-Run a simple HTTP server:
+3. Run a simple HTTP server:
 
 ```sh
 python -m http.server 8000
 ```
 
-In a browser, open:
+4. Then in a browser, open:
 
 - [http://localhost:8000/?url=&book=guide](http://localhost:8000/?url=&book=guide)
 - [http://localhost:8000/?url=&book=reference](http://localhost:8000/?url=&book=reference)
 - [http://localhost:8000/?url=&book=blog](http://localhost:8000/?url=&book=blog)
 - [http://localhost:8000/?url=&book=automobile](http://localhost:8000/?url=&book=automobile)
 
 
-## Run the unit tests
+## For running the unit tests, the following steps should be followed:
 
-Install the pep8 module:
+1.Install the pep8 module:
```

Comment:
Space is missing here as well.

Reply:
I have tried to add the spaces over here as well . Sorry for the trouble.